Platform/obc/hal/comms_hal.c

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (C) 2015  University of Alberta
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ */
+/**
+ * @file comms_hal.c
+ * @author Arash Yazdani
+ * @date 2020-10-01
+ */
+
+/*Once the hardware is connected, toggle this Macro*/
+#define Stubbed = 1;
+
+#include "obc_hal.h"
+#include "comms_hal.h"
+#include <stddef.h>
+#include <FreeRTOS.h>
+#include <csp/csp.h>
+#include <csp/csp_endian.h>
```

Comment:
Might not be using the `csp` headers, also the `FreeRTOS` functions, until we start using mutexes in here